### PR TITLE
Do not log binary data in "multipart/form-data" request

### PR DESCRIPTION
**Pull request by tarik-k**

### CODE REVIEW

**Comment by tclancy** [comment]
It looks like that PR fails the unit tests on any version of Python 3

**Comment by tarik-k** [comment]
I fixed test, now everything is ok.

File: request_logging/middleware.py [hunk 1]
```diff
@@ -46,7 +48,7 @@ def _log(self, level, msg, colour):
 
 class LoggingMiddleware(MiddlewareMixin):
     def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+        super(LoggingMiddleware, self).__init__(*args, **kwargs)
```
Comment (tclancy) [inline request_logging/middleware.py]: Good catch, thanks.

File: request_logging/middleware.py [hunk 2]
```diff
@@ -90,6 +101,19 @@ def process_response(self, request, response):
 
         return response
 
+    def _log_multipart(self, body):
```
Comment (tclancy) [inline request_logging/middleware.py]: Can you add a docstring explaining what this does? While you're in here, can you also modify `_chunked_to_max` to simply return `msg[:self.max_body_length]` without bothering to check the whole length for performance reasons? I know we lose the `...` indicating there was more data, but I think the tradeoff is worth it.

Comment (tarik-k) [reply request_logging/middleware.py]: Done

**Comment by tclancy** [comment]
One last change: you need to bump the version in `setup.py`

**Pull request merged**